src/js/stores/DCOSStore.js

```diff
@@ -71,6 +79,13 @@ class DCOSStore extends EventEmitter {
   }
 
   /**
+   * Fetch launch queue from Marathon
+   */
+  fetchQueue() {
```

Comment:
I would prefer a more descriptive name here, so we could get rid of the distracting comment.
Maybe `fetchMarathonLaunchQueue()`.
As always just a suggestion.